### PR TITLE
Homework 4 is done

### PR DESCRIPTION
**Pull request by serafimovmax**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
 *.DS_Store
 */.DS_Store
-**/.DS_Store
\ No newline at end of file
+**/.DS_Store
+/.allure
+.allure
+/allure-results
+vcs.xml
+out/
```
Comment (komyagin) [inline .gitignore:12]: Add a new blank line at the end of the file

File: src/test/java/com/epam/tc/hw4/BaseHomePageTest.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        //1. Open test site by URL
+        loginPage.openPage();
+
+        //2. Assert Browser title
+        softAssert.assertTrue(loginPage.getPageTitle().contains(TITLE_HOME_PAGE.get()));
+
+        //3. Perform login
+        mainPage = loginPage.login(someUser);
+
+
```
Comment (komyagin) [inline src/test/java/com/epam/tc/hw4/BaseHomePageTest.java:42]: Remove one blank line

File: src/test/java/com/epam/tc/hw4/PageElements.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
+package com.epam.tc.hw4;
+
+import java.util.List;
+
+public class PageElements {
+
+
```
Comment (komyagin) [inline src/test/java/com/epam/tc/hw4/PageElements.java:7]: Remove a blank line

File: src/test/java/com/epam/tc/hw4/page/DifferentElementsPage.java [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    @FindBy(className = "colors")
+    private WebElement dropdownColors;
+
+    @FindBy(xpath = "//option[text()='Yellow']")
+    private WebElement yellowColor;
+
+    @FindBy(xpath = "//ul[@class='panel-body-list logs']/li")
+    private List<WebElement> logsComponent;
+
+
```
Comment (komyagin) [inline src/test/java/com/epam/tc/hw4/page/DifferentElementsPage.java:37]: Remove blank line

File: src/test/java/com/epam/tc/hw4/page/MainPage.java [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+    @FindBy(id = "frame")
+    private WebElement iframe;
+
+    @FindBy(id = "frame-button")
+    private WebElement frameButton;
+
+    @FindBy(xpath = "//ul[@class='sidebar-menu left']/li")
+    private List<WebElement> leftMenu;
+
+
```
Comment (komyagin) [inline src/test/java/com/epam/tc/hw4/page/MainPage.java:34]: remove blank line

**Review by komyagin — CHANGES_REQUESTED** [review]

File: src/test/java/com/epam/tc/hw4/page/MainPage.java (hunk 5 above)
Comment (serafimovmax) [reply src/test/java/com/epam/tc/hw4/page/MainPage.java:34]: done

File: src/test/java/com/epam/tc/hw4/page/DifferentElementsPage.java (hunk 4 above)
Comment (serafimovmax) [reply src/test/java/com/epam/tc/hw4/page/DifferentElementsPage.java:37]: done

File: src/test/java/com/epam/tc/hw4/PageElements.java (hunk 3 above)
Comment (serafimovmax) [reply src/test/java/com/epam/tc/hw4/PageElements.java:7]: done

File: src/test/java/com/epam/tc/hw4/BaseHomePageTest.java (hunk 2 above)
Comment (serafimovmax) [reply src/test/java/com/epam/tc/hw4/BaseHomePageTest.java:42]: done

File: .gitignore (hunk 1 above)
Comment (serafimovmax) [reply .gitignore:12]: done

**Review by komyagin — APPROVED** [review]

**Pull request merged**